compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -529,3 +529,18 @@ function parseIntLiteralExpr(Tokenizer tok) returns IntLiteralExpr|err:Syntax {
 public function intFromIntLiteral(IntLiteralBase base, string digits) returns int|error {
     return base == 10 ? int:fromString(digits) : int:fromHexString(digits);
 }
+
+function parseMethodCallExpr(Tokenizer tok) returns MethodCallExpr|err:Syntax {
```

Comment:
There is code above that deals with parsing method call expressions. This is not what this code is doing. Look at `tok.expect(";")`. Put this back into parseMethodCallStmt and look at it again.

Reply:
Fixed